graphics/tinygl/clip.cpp

```diff
@@ -226,13 +226,12 @@ float(*clip_proc[6])(Vector4 *, Vector4 *, Vector4 *) =  {
 
 static inline void updateTmp(GLContext *c, GLVertex *q, GLVertex *p0, GLVertex *p1, float t) {
 	if (c->current_shade_model == TGL_SMOOTH) {
-		float a = q->color.W;
 		q->color = p0->color + (p1->color - p0->color) * t;
-		q->color.W = a;
```

Comment:
BTW, doesn't C++ create 2 (or maybe even 3) intermediate Vector4 instances in the
```c++
q->color = p0->color + (p1->color - p0->color) * t;
```
line ? One for `-`, one for `*`, and one for `+`. I do not know if the last one is copied to existing `q->color` Vector4 instance or if it replaces it.